test/conversations/Conversation.test.ts

```diff
@@ -385,134 +387,140 @@ describe('conversation', () => {
       for (let i = 0; i < 5; i++) {
         await bobConvo.send('hi')
       }
+      await sleep(100)
       const messages = await bobConvo.messages({ limit: 2 })
       expect(messages).toHaveLength(2)
     })
-  })
 
-  it('conversation filtering', async () => {
```

Comment:
From here down the code isn't changing, it's just moving into the v2 suite scope. It was at the top level before which had it effectively running with v1 behavior, but these are largely v2 tests, so I think they should run in v2 scope.